raiden-dapp/src/components/dialogs/UdcWithdrawalDialog.vue

```diff
@@ -143,14 +143,19 @@ export default class UdcWithdrawalDialog extends Vue {
       await this.$raiden.planUdcWithdraw(this.withdrawAmount);
       this.done();
     } catch (e) {
-      this.error = e.message;
+      this.error = e;
```

Comment:
Nitpick: `error` is still declared as a `string`, now it could be `any`

Reply:
Thanks fixing it :+1:

Reply:
Thanks fixed it :+1: